scripts/node/dictionary/transforms.js

```diff
@@ -98,43 +98,6 @@ const toRgbComponents = (prop) => {
   return `${r}, ${g}, ${b}`;
 };
 
-/**
- * @param {Object} prop style-dictionary prop
- * @returns {Object} object representing material UI palette color values
- */
-const toMaterialPalette = (prop) => {
-  const tintKeys = [400, 300, 200, 100, 50];
-  const shadeKeys = [600, 700, 800, 900];
-
-  const flattenReducer = (acc, curr) => {
-    acc[curr.key] = curr.color;
-    return acc;
-  };
-
-  const tints = tintKeys
-    .map((key, i) => ({
-      key,
-      color: materialTint(prop.value, i + 1),
```

Comment:
now that these material color functions are no longer used, you can probably get rid of a bunch of stuff in `util/color.js`